### PR TITLE
#13 Materials administration

### PR DESCRIPTION
**Pull request by luisferyimi19**
Creation of CRUD, model, validations and views.

### CODE REVIEW

File: app/controllers/materials_controller.rb [hunk 1]
```diff
@@ -1,49 +1,85 @@
...
+  # GET /materials
+  # GET /materials.json
+  def index
+    @q = Material.ransack(search_material_params)
+    @materials = @q.result
+    @materials = @materials.page(params[:page])
+  end
+
+  # GET /materials/new
+  def new
```
Comment (Chofo2003) [inline app/controllers/materials_controller.rb]: Unnecessary method

File: app/controllers/materials_controller.rb [hunk 2]
```diff
@@ -1,49 +1,85 @@
...
+    @material = Material.new
+  end
+
+  # GET /materials/1/edit
+  def edit
```
Comment (Chofo2003) [inline app/controllers/materials_controller.rb]: Unnecessary method

File: app/views/materials/_form.html.haml [hunk 3]
```diff
@@ -1,27 +1,20 @@
...
-                %option
-                    %b Metro
-                %option
-                    %b Caja
-                %option
-                    %b Bobina
-                %option
-                    %b Unidad
+  = form_for @material do |f|
+    -# if @material.errors.any?
```
Comment (Chofo2003) [inline app/views/materials/_form.html.haml]: Delete code commented.

Comment (luisferyimi19) [reply app/views/materials/_form.html.haml]: commented code removed.

File: app/controllers/materials_controller.rb (hunk 2 above)
Comment (luisferyimi19) [reply app/controllers/materials_controller.rb]: Unnecessary method removed

**Pull request merged**